client/app/bundles/Zoon/components/WorkflowDiagram.jsx

```diff
@@ -1,20 +1,28 @@
 import React from "react"
 import Reorder from "react-reorder"
+import classnames from "classnames"
+
+import {upFirstLetter} from "../utils"
 
 const families = ["occurrence", "covariate", "process", "model", "output"]
+export const allCompositionTypes = ["list", "chain", "replicate"]
 
-const Switch = ({ id, checked, onChange }) => (
-  <span className="switch tiny" style={{"margin": "0em 0.5em"}}>
-    <input
-      type="checkbox"
-      className="switch-input"
-      id={id}
-      checked={checked}
-      onChange={onChange}
-    />
-    <label className="switch-paddle" htmlFor={id} />
-  </span>
-)
+const Switch = ({ value, onChange }) => <div className="module-list__switch">
```

Comment:
Other functional components in this project have a different style:

    const SelectedItem = ({ item, sharedProps }) => (
      <el>
        <el2></el2>
      </el>
    );

instead of 

    const SelectedItem = ({ item, sharedProps }) => <el>
      <el2></el2>
    </el>;